### DISCUSSION TITLE
Possibility to control 2nd connector on "Enelion Wallbox Duo Power" charger?

### DISCUSSION THREAD

**Discussion by blidegn** [lbbrhzn/ocpp#792] [Q&A]
I have an "Enelion Wallbox Duo Power" charger with two connectors.

Currently the integration will activate the first connector when using the "Charge control" switch. 
Will it be possible to have a switch for each connector so both of them can be controlled via this integration?
It seems like the integration recognises two connectors when looking at the Diagnostic section. 
See screen dumps below:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/54290094/224557265-789531c3-d0ee-4c4e-b2d9-1506af8e58b8.png">
<img width="259" alt="image" src="https://user-images.githubusercontent.com/54290094/224557182-d28b8366-56c4-45cf-9f26-7fd1289d18ae.png">
<img width="236" alt="image" src="https://user-images.githubusercontent.com/54290094/224557204-e9b0a0ca-f3dc-4496-820d-59c7dbc22a3a.png">

Best regards

**Comment by ernst77** [comment]
We have a charger with two connectors as well, would be nice to have support for both of them. As well separate sensors for each of them.
